Makefile

```diff
@@ -41,24 +43,34 @@ endif
 .PHONY: all
 all: check-style test dist
 
+## Propagates plugin manifest information into the server/ and webapp/ folders.
+.PHONY: apply
+apply:
+	./build/bin/manifest apply
+
+## Install go tools
+install-go-tools:
+	@echo Installing go tools
+	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.1
```

Comment:
Do we actually need to install the binary? Or is 
```suggestion
	$(GO) run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.1
```
enough?

Reply:
Oh, that would be convenient! But unfortunately much slower it seems?

![CleanShot 2024-01-23 at 09 47 36@2x](https://github.com/mattermost/mattermost-plugin-starter-template/assets/1023171/1ce1bf15-0faa-4622-844a-ae9a41ae835b)

Reply:
Well, the added duration comes from checking the remote go modules DB. The increase is similar running `go install`.

One downside of `go run` is, that it errors if the machine is offline.

Reply:
Ah, you're right: I wasn't comparing apples to apples as I thought!